src/MGA/Check/UnreachablePath.php

```diff
@@ -29,8 +29,11 @@ class UnreachablePath
         'admin',
         'app/etc/local.xml',
         'composer.json',
+        'cron.php',
```

Comment:
This is exposed on purpose to those without the ability to configure cron jobs can still run Magento's cron externally.  It doesn't reveal anything worth while either.